"0",""
"0","## Get Data"
"0","import duckdb"
"2","Traceback (most recent call last):
  File ""<string>"", line 1, in <module>
  File ""[90m]8;line = 119;file://C:\Users\C25Chase.Powers\AppData\Local\R\cache\R\renv\cache\v5\R-4.2\x86_64-w64-mingw32\reticulate\1.35.0\90be16b53b955990db4aa355c03d85eb\reticulate\python\rpytools\loader.pyC:\Users\C25Chase.Powers\AppData\Local\R\cache\R\renv\cache\v5\R-4.2\x86_64-w64-mingw32\reticulate\1.35.0\90be16b53b955990db4aa355c03d85eb\reticulate\python\rpytools\loader.py]8;;[39m"", line 119, in _find_and_load_hook
    return _run_hook(name, _hook)
  File ""[90m]8;line = 93;file://C:\Users\C25Chase.Powers\AppData\Local\R\cache\R\renv\cache\v5\R-4.2\x86_64-w64-mingw32\reticulate\1.35.0\90be16b53b955990db4aa355c03d85eb\reticulate\python\rpytools\loader.pyC:\Users\C25Chase.Powers\AppData\Local\R\cache\R\renv\cache\v5\R-4.2\x86_64-w64-mingw32\reticulate\1.35.0\90be16b53b955990db4aa355c03d85eb\reticulate\python\rpytools\loader.py]8;;[39m"", line 93, in _run_hook
    module = hook()
  File ""[90m]8;line = 117;file://C:\Users\C25Chase.Powers\AppData\Local\R\cache\R\renv\cache\v5\R-4.2\x86_64-w64-mingw32\reticulate\1.35.0\90be16b53b955990db4aa355c03d85eb\reticulate\python\rpytools\loader.pyC:\Users\C25Chase.Powers\AppData\Local\R\cache\R\renv\cache\v5\R-4.2\x86_64-w64-mingw32\reticulate\1.35.0\90be16b53b955990db4aa355c03d85eb\reticulate\python\rpytools\loader.py]8;;[39m"", line 117, in _hook
    return _find_and_load(name, import_)
ModuleNotFoundError: No module named 'duckdb'
"
"0","con = duckdb.connect('my-db.duckdb')"
"2","Traceback (most recent call last):
  File ""<string>"", line 1, in <module>
NameError: name 'duckdb' is not defined
"
"0","df = con.execute(""SELECT * FROM penguins"").fetchdf().dropna()"
"2","Traceback (most recent call last):
  File ""<string>"", line 1, in <module>
NameError: name 'con' is not defined
"
"0","con.close()"
"2","Traceback (most recent call last):
  File ""<string>"", line 1, in <module>
NameError: name 'con' is not defined
"
"0","df.head(3)"
"2","Traceback (most recent call last):
  File ""<string>"", line 1, in <module>
NameError: name 'df' is not defined
"
